lexicons/app/bsky/graph/getRelationships.json

```diff
@@ -0,0 +1,85 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.getRelationships",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Enumerates public relationships between one account, and a list of other accounts",
+      "parameters": {
+        "type": "params",
+        "required": ["actor"],
+        "properties": {
+          "actor": { "type": "string", "format": "at-identifier" },
+          "others": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "at-identifier"
+            }
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["relationships"],
+          "properties": {
+            "actor": { "type": "string", "format": "did" },
+            "relationships": {
+              "type": "array",
+              "items": {
+                "type": "union",
+                "refs": ["#relationship", "app.bsky.graph.defs#notFoundActor"]
+              }
+            }
+          }
+        }
+      },
+      "errors": [
+        {
+          "name": "ActorNotFound",
+          "description": "the primary actor at-identifier could not be resolved"
+        }
+      ]
+    },
+    "relationship": {
+      "type": "object",
+      "required": ["did"],
+      "properties": {
+        "did": { "type": "string", "format": "did" },
+        "handle": { "type": "string", "format": "handle" },
+        "blocks": {
+          "type": "string",
+          "format": "at-uri",
+          "description": "if the actor blocks this DID, this is the AT-URI of the block record"
+        },
+        "blocks_via_list": {
```

Comment:
In viewer state it's "blocked" and "blocking".  I wouldn't claim it's perfect, but maybe we should be consistent with it?

Reply:
mmm, yeah, always get confused by those terms, but consistency is best; updated